py/tests/plane_csc.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+
+import pykms
+
+card = pykms.Card()
+res = pykms.ResourceManager(card)
+conn = res.reserve_connector("HDMI")
+crtc = res.reserve_crtc(conn)
+mode = conn.get_default_mode()
+modeb = mode.to_blob(card)
+plane = res.reserve_generic_plane(crtc, pykms.PixelFormat.UYVY)
+#plane = res.reserve_generic_plane(crtc, pykms.PixelFormat.Undefined)
+
+print("Got plane %d %d" % (plane.idx, plane.id))
+
+fb = pykms.DumbFramebuffer(card, mode.hdisplay, mode.vdisplay, "UYVY");
+pykms.draw_test_pattern(fb);
+
+req = pykms.AtomicReq(card)
+req.add(conn, "CRTC_ID", crtc.id)
+req.add(crtc, {"ACTIVE": 1,
+               "MODE_ID": modeb.id})
+
+input("Press enter to enable crtc idx %d at %s" % (crtc.idx, conn.fullname))
+r = req.commit_sync(allow_modeset = True)
+
+input("Press enter to enable plane idx %d at %s" % (plane.idx, conn.fullname))
+
+req = pykms.AtomicReq(card)
+req.add(plane, {"FB_ID": fb.id,
+                "CRTC_ID": crtc.id,
+                "SRC_X": 0 << 16,
+                "SRC_Y": 0 << 16,
+                "SRC_W": fb.width << 16,
+                "SRC_H": fb.height << 16,
+                "CRTC_X": 0,
+                "CRTC_Y": 0,
+                "CRTC_W": fb.width,
+                "CRTC_H": fb.height,
+                "zorder": 0})
+r = req.commit_sync()
+print("Plane enable request returned %d\n" % r)
+
+yuv_types = []
+yuv_types.append(pykms.YUVType.BT601_Lim)
+yuv_types.append(pykms.YUVType.BT601_Full)
+yuv_types.append(pykms.YUVType.BT709_Lim)
+yuv_types.append(pykms.YUVType.BT709_Full)
+
+for i in range(0, 2):
+    for j in range(0, 2):
+        input("press enter to for encoding %d range %d\n" % (i, j))
+
+        plane.set_prop("COLOR_ENCODING", i);
+        plane.set_prop("COLOR_RANGE", j);
+
```

Comment:
Probably doesn't matter much, but I think you should use atomic commit here, with the two properties set at once.